dev-client/src/model/soilId/soilIdPlaceholders.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright © 2024 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+/* To be replaced with actual integration w/ soilId schema */
```

Comment:
At the time this PR was opened, the Client Shared repo hadn't received the PR which adds these types / the API endpoints, so I added temporary data to fill in for it.

Reply:
i'm happy for you to merge as is and replace in a follow-on PR, or rebase this on main and use the client-shared types, whatever seems easier to you!

Reply:
I'll handle it in the upcoming [Integrate frontend with US Soils Dataset](https://github.com/techmatters/terraso-product/issues/653), in the interest of moving this one forward.